blueprints/in-repo-addon/index.js

```diff
@@ -69,9 +69,12 @@ module.exports = {
       let newPathIndex = paths.indexOf(newPath);
       if (newPathIndex > -1) {
         paths.splice(newPathIndex, 1);
-        if (paths.length === 0) {
-          delete contents['ember-addon']['paths'];
-        }
+      }
+      if (paths.length === 0) {
```

Comment:
Moved this if outside of the previous one. This will also clean up the `paths` array if the user already manually deleted the path.

Reply:
> This will also clean up the paths array in case the user already manually deleted the in-repo addon's path.

Should we test this case too?

Reply:
Yup! Test added.